builtin/logical/pki/backend_test.go

```diff
@@ -2126,12 +2128,31 @@ func TestBackend_SignVerbatim(t *testing.T) {
 			"ttl": "12h",
 		},
 	})
-	if resp != nil && !resp.IsError() {
-		t.Fatalf("sign-verbatim signed too-large-ttl'd CSR: %#v", *resp)
+	if err != nil {
+		t.Fatal(err)
+	}
+	if resp != nil && resp.IsError() {
+		t.Fatalf(resp.Error().Error())
```

Comment:
Is there a need for double `Error()` here?

Reply:
maybe if I switch it to t.Fatal -- t.Fatalf expects a string, and resp.Error() returns an `error`